bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/DaikinWebTargets.java

```diff
@@ -136,7 +176,14 @@ private String invoke(String uri, Map<String, String> params) throws DaikinCommu
         String response;
         synchronized (this) {
             try {
-                response = HttpUtil.executeUrl("GET", uriWithParams, TIMEOUT_MS);
+                if (httpClient != null) {
+                    response = executeUrl(uriWithParams);
+                } else {
+                    // a fall back method
+                    response = HttpUtil.executeUrl("GET", uriWithParams, TIMEOUT_MS);
+                }
+            } catch (DaikinCommunicationException ex) {
+                throw ex;
```

Comment:
Does DaikinCommunicationException extend an `IOException`? If not then you don't need to catch it here.

Reply:
Yes it does